docs/source/examples/workspace/PinFile.dummy.yml

```diff
@@ -3,6 +3,6 @@ dummy-new:
   topology: dummy-new.yml
   layout: dummy-new.yml
 
-dummy-topo:
-  topology: dummy-new.yml
-  layout: dummy.yml
+#dummy-topo:
+#  topology: dummy-new.yml
+#  layout: dummy.yml
```

Comment:
I use this for the linchpin container builds, please don't remove it.

Reply:
I just used this for testing and accidentally committed it.  I'll revert it